app/app_xdg.go

```diff
@@ -58,8 +58,12 @@ func findFreedestktopColorScheme() fyne.ThemeVariant {
 		return theme.VariantDark
 	}
 
+	// See: https://github.com/flatpak/xdg-desktop-portal/blob/1.16.0/data/org.freedesktop.impl.portal.Settings.xml#L32-L46
+	// 0: No preference
+	// 1: Prefer dark appearance
+	// 2: Prefer light appearance
 	switch value {
-	case 0:
+	case 2:
 		return theme.VariantLight
 	default:
 		return theme.VariantDark
```

Comment:
I'd suggest that we remove the switch and just do it like this:
```go
if value == 1 {
    return theme.VariantDark
}

return theme.VariantLight
```

This should make sure that we handle 2 for light and also support switching on Gnome.

Reply:
Yes I guess this is probably the best we can do - but I feel it would be good to know if we are working around a GNOME bug (that will be resolved) or if the implement this standard only partially?

Reply:
I honestly don't know but I wouldn't want to break support for one of the most used/popular desktops either way.

Reply:
Since this has been the behaviours since Gnome 42, I would think it is their desired behaviour, but I honestly don't know. Wouldn't want to break support on one of the most popular desktops though.

Reply:
Indeed, it depends what theme we want show for the default value `0`.
It looks like gnome `42` will use the light one where no preference as set (i.e. fresh install), so the suggestion above makes sense.
Tested on Ubuntu 22.04 (Gnome 42.5) and works as expected in both cases.
It seems the only thing we need to decide is which theme apply as default when the portal preference is supported and if the default theme in case of error is the same or always black.

Reply:
I am using Gnome 43 and it behaves just like 42. I'd say that we default to dark in the case of an error as it would be most like how we do on other platforms.

Reply:
Make sense, updated in 0dffef90e81abc620eb1eb7ba8b49fa7b709204b to support Gnome's default